Contuity/Views/Jot/Read/ReadJotViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ReadJotViewController.swift
+//  Contuity
+//
+//  Created by Reza Akhtar on 18.10.18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ReadJotViewController: UIViewController {
+    var jotID: Int = 1 // TODO: this needs to be set to the given jotID in a constructor
+    var presenter: ReadJotPresenter = ReadJotPresenter()
+    
+    /// MARK - properties
+    @IBOutlet private (set) var readJotTextView: UITextView!
+    @IBOutlet private (set) var backButton: UIButton!
+    @IBOutlet private (set) var editButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        readJotTextView.delegate = self
+        presenter.attachView(self)
+        presenter.setText(jotID: jotID)
+        prettify()
+        navigationItem.title = "Read Jot"
+    }
+    @IBAction func editButtonTapped(_ sender: UIButton) {
+        let editWriteJotVC = WriteJotViewController()
+        editWriteJotVC.presenter.setJotID(givenID: jotID)
+        editWriteJotVC.presenter.update = true
+        navigationController?.pushViewController(editWriteJotVC, animated: true)
+    }
+    @IBAction func backButtonTapped(_ sender: UIButton) {
+        navigationController?.popViewController(animated: true)
+    }
+}
+
+extension ReadJotViewController: UITextViewDelegate {
+    func textViewDidChange(_ textView: UITextView) {
+        presenter.text = textView.text
```

Comment:
Will this ever even get called?? When will the text view change in the ReadJotVC? I might just be missing something here

If it's not getting called, please:
- remove the UITextViewDelegate logic
- remove the `var text: String` from the Presenter and just make `setText(jotID: Int)` to something like `getText(from jotId: Int) -> Strings` which returns the text for the viewcontroller to display

Reply:
Good call, it wasn't getting called. That was just there when I was trying to mimic what you had to see why mine wasn't working. It's deleted now.